front/public/cards/wsi.json

```diff
@@ -0,0 +1,27 @@
+{
+    "ribbonMessage": "We're excited to announce our new whole slide imaging viewer!",
+    "ppcalls": [
+        {
+            "runargs": {
+                "noheader": true,
+                "mass": {
+                    "state": {
+                        "dslabel": "TermdbTest",
+                        "genome": "hg38-test",
+                        "nav": {
+                            "header_mode": "hidden"
+                        },
+                        "sample_id": "B-T87L964D",
```

Comment:
if sample id is visible on app drawer, should change to `TCGA-22-1017` which is the correct id for the demo image